<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/img/logo yaz覺s覺z.png</normaloff>../assets/img/logo yaz覺s覺z.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,13">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="aaaa" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../assets/img/Quark(BeyazLogo).png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>190</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../assets/img/interface.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>150</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="userGuideButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../assets/img/icons8-open-book-50.png</normaloff>../assets/img/icons8-open-book-50.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="infoButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../assets/img/icons8-info-50.png</normaloff>../assets/img/icons8-info-50.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="mainLayout" stretch="3,1">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="interfaceLayout" stretch="1,5">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="optionsLayout" stretch="1,2,3,4">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <layout class="QVBoxLayout" name="comPortLayout" stretch="1,1,1">
              <item>
               <widget class="QComboBox" name="comPorts">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="refreshButton">
                <property name="text">
                 <string>Refresh</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="connectButton">
                <property name="text">
                 <string>Connect</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="gaugeLayout" stretch="5">
              <item alignment="Qt::AlignBottom">
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                  <italic>false</italic>
                  <bold>true</bold>
                  <strikeout>false</strikeout>
                  <kerning>false</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>Temperature</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="setConfig_ReadingLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>15</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="setConfigLayout" stretch="1,1">
                <property name="spacing">
                 <number>15</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Set Configurations</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,0,0">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QComboBox" name="configsComboBox">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="addConfigButton">
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="addToolButton">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="HVReadingLayout" stretch="0,0">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item alignment="Qt::AlignBottom">
                 <widget class="QLabel" name="HVReadingLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>High Voltage Reading:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <property name="spacing">
                   <number>8</number>
                  </property>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="settingsConfigLayout" stretch="0,0">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QLabel" name="minVoltageLabel">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Min High Voltage</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="maxVoltageLabel">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Max High Voltage</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="minTempLabel">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Min Temp</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="maxTempLabel">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Max Temp</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QSlider" name="minVoltageSlider">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="font">
                     <font>
                      <family>Segoe Script</family>
                      <hintingpreference>PreferVerticalHinting</hintingpreference>
                     </font>
                    </property>
                    <property name="contextMenuPolicy">
                     <enum>Qt::DefaultContextMenu</enum>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="maxVoltageSlider">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="minTempSlider">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="maxTempSlider">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QDoubleSpinBox" name="minVoltageSpinBox"/>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="maxVoltageSpinBox"/>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="minTempSpinBox"/>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="maxTempSpinBox"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="configureButton">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="toolTipDuration">
                   <number>-1</number>
                  </property>
                  <property name="text">
                   <string>Send Configuration</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="startProcessButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Start Process</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="graphLayout">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <layout class="QVBoxLayout" name="tempLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetFixedSize</enum>
              </property>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="voltageeeeLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetFixedSize</enum>
              </property>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="terminal" stretch="18,1,1">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QTextEdit" name="terminalTextEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="acceptRichText">
             <bool>false</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sendLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendCommandButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Send Command</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiondfsdfsdfs">
   <property name="text">
    <string>dfsdfsdfs</string>
   </property>
  </action>
  <action name="actiondasdasd">
   <property name="text">
    <string>dasdasd羹</string>
   </property>
  </action>
  <action name="actiondfsdfsdf">
   <property name="text">
    <string>dfsdfsdf</string>
   </property>
  </action>
  <action name="actiondasasSD">
   <property name="text">
    <string>dasasSD</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionTest_1_1">
   <property name="text">
    <string>Test 1-1</string>
   </property>
  </action>
  <action name="actionTest1_2">
   <property name="text">
    <string>Test1-2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
